samza-rest/src/test/java/org/apache/samza/rest/TestSamzaRestService.java

```diff
@@ -22,12 +22,15 @@
 import junit.framework.TestCase;
 import org.apache.samza.metrics.MetricsReporter;
 import org.apache.samza.metrics.ReadableMetricsRegistry;
-import org.eclipse.jetty.server.Connector;
+import org.eclipse.jetty.server.NetworkConnector;
 import org.eclipse.jetty.server.Server;
 import org.eclipse.jetty.servlet.ServletContextHandler;
+// Temporary fix to fix broken master. Long term fix is to mock jetty objects properly.
```

Comment:
It would be nice to open a JIRA and put the JIRA number here. You can open the JIRA later.